app/ui/shared/widgets/overview/summaries/star.jsx

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import React, { Component, PropTypes } from 'react';
+import PureRenderMixin from 'react-addons-pure-render-mixin';
+
+import Btn from '../../../../shared/widgets/btn';
+
+const STAR_IMAGES = {
+  full: 'glyph-ratings-full.svg',
+  half: 'glyph-ratings-half.svg',
+  empty: 'glyph-ratings-empty.svg',
+};
+
+class Star extends Component {
+  constructor(props) {
+    super(props);
+    this.shouldComponentUpdate = PureRenderMixin.shouldComponentUpdate.bind(this);
```

Comment:
From https://facebook.github.io/react/docs/pure-render-mixin.html:

> Furthermore, shouldComponentUpdate skips updates for the whole component subtree. Make sure all the children components are also "pure".

Is it safe to assume that Btn is pure?  If so, why is it using isEqual in the previous commit?  https://github.com/mozilla/tofino/pull/1047/commits/04261c74ec00a77ea263b3ff8e2fdd0a413170a6#diff-1f66465f034f1cd156f8ff2bc8be49b3R48